src/AD419/Logging/SerilogHttpContextEnricher.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Concurrent;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace AD419.Logging
+{
+    public class SerilogHttpContextEnricher : ILogEventEnricher
```

Comment:
we use this so much we should look into making a simple nuget package like other enrichers.  Future something to think about when we have time.